app/screens/about/about.js

```diff
@@ -228,11 +236,34 @@ export default class About extends PureComponent {
                         <FormattedText
                             id='mobile.about.copyright'
                             defaultMessage='Copyright 2015-{currentYear} Mattermost, Inc. All rights reserved'
-                            style={style.footerText}
+                            style={[style.footerText, {marginBottom: 0}]}
                             values={{
                                 currentYear: new Date().getFullYear(),
                             }}
                         />
+                        <FormattedText
+                            id='about.tos_privacy_text'
+                            defaultMessage='{tos} - {privacy}'
+                            style={style.footerText}
+                            values={{
+                                tos: (
+                                    <FormattedText
+                                        id='mobile.tos_link'
+                                        defaultMessage='Terms of Service'
+                                        style={style.noticeLink}
+                                        onPress={this.handleTermsOfService}
+                                    />
+                                ),
+                                privacy: (
+                                    <FormattedText
+                                        id='mobile.privacy_link'
+                                        defaultMessage='Privacy Policy'
+                                        style={[style.noticeLink, {marginLeft: 5}]}
```

Comment:
I don't think this margin actually does anything due to how RN handles styles for nested texts. If it isn't needed, can you remove it? If it is needed, can you move it to the StyleSheet as well?